README.md

```diff
@@ -35,18 +35,29 @@ jobs:
     steps:
       - uses: knawat/github-actions-deploy-moleculer@main
         with: 
-          BASE_URL: 'api.knawat.io'
-          DNS_ZONE_NAME: 'knawat-io'
-          docker-registry: 'gcr.io'
-          IMAGE_REPO_NAME: 'api'
+          #APPLICATION VARIABLE
+          SERVICES: {api,products}
+          SERVICEDIR: build/services
+          MOLECULER_APM_ENABLE: 1
+          AGENT_TOKEN: ${{ secrets.AGENT_TOKEN }}
+          AGENT_APIKEY: ${{ secrets.AGENT_APIKEY }}
+          TRANSPORTER: tcp
+          MONGO_URI: '${{ secrets.MONGO_URI }}' //should include it between ''
+          BASE_URL: api.example.com
```

Comment:
if this can be changed to a more reflecting domain name for easy understanding will be better.